exercises/beer-song/beer-song.js

```diff
@@ -3,12 +3,6 @@
 // convenience to get you started writing code faster.
 //
 
-export class BeerSong {
-  static verse() {
-    throw new Error("Remove this statement and implement this function");
-  }
-
-  static sing() {
-    throw new Error("Remove this statement and implement this function");
-  }
-}
+export const recite = () => {
```

Comment:
Can you add the arguments (named correctly)? It's not really clear from the tests what the arguments _are_. Another idea is to add a HINT insert file and regenerate the readme.

Reply:
For more info on hint file: https://github.com/exercism/configlet#hints

Reply:
I've added the arguments. For the moment it seems enough.